.travis.yml

```diff
@@ -97,7 +103,34 @@ deploy:
       tags: true
       repo: coveo/search-ui
       all_branches: true
-      condition: "! $TRAVIS_TAG =~ .*beta.*"
+      condition: "$IS_RELEASE == true"
+  - provider: s3
+    access_key_id: AKIAYKDJLZIT462WODCI
+    secret_access_key:
+      secure: sT8isIiaAIg/Ic7x8cCD3anwCEHAyzG/IWkngmkf5c128NEwUIUdmsQJ1hi4Bw84zk1OOb3id+yXjeaTuKDgIkxe/jk3e2xZ7xElfLNTOln6QLkgsYpKFtGHiKjROOHOb5+Es6L1mxpHWbXH9ebmEhRduUw/BeHWW2MfZmfb434DgvatViRm8DOeROeqlYnbCda5ZELuT5SdwEs/CbExaiSL0WKVoTvcxbe+At4uVnTuc6SYyOvRoblx/q6Ra6mdRipluMs7Go93eu/Oqo7cZ2akrNSi/WBXGqL/wXxe2pYsziSABWjQaqWXD/jKte1ZBx7OXssn88DUKat/RrL7Ak30WHzgY/goiNV+EPT/wtNYWHP3suAzzGqNNMhsPx0lSI7vSUga72J3hV1TOsX9ORosiRm1c2fnmNB9vJdH7yAX7lWpqH2XuOrldVvvTGazSJYx93eBukDViFCZ6egeb+gMT8VQ38uitwd9BZaMhycoujZgSjhx7JnbHUoUdXkdT3XBNV7fINTozWCNJW81iJN+JuLZzneJ/CbjmjkfGcVoFFPCD9rGSj8DbgvWpt8rkwI4ZfHTNOgO9iWgSklpk9r2wgM7Z87V2Zpu8UX/QCFxy8IREtJUo6X9YG9nBoy21wUa9zia2SWpideur9cXb6U0+q84onHVlA0Ehp/1OYs=
+    bucket: coveo-nprod-binaries
+    local-dir: bin
+    upload-dir: proda/StaticCDN/searchui/nightly
+    acl: public_read
+    skip_cleanup: true
+    on:
+      tags: true
+      repo: coveo/search-ui
+      all_branches: true
+      condition: "$IS_BETA == true"
+  - provider: s3
+    access_key_id: AKIAYKDJLZIT462WODCI
+    secret_access_key:
+      secure: sT8isIiaAIg/Ic7x8cCD3anwCEHAyzG/IWkngmkf5c128NEwUIUdmsQJ1hi4Bw84zk1OOb3id+yXjeaTuKDgIkxe/jk3e2xZ7xElfLNTOln6QLkgsYpKFtGHiKjROOHOb5+Es6L1mxpHWbXH9ebmEhRduUw/BeHWW2MfZmfb434DgvatViRm8DOeROeqlYnbCda5ZELuT5SdwEs/CbExaiSL0WKVoTvcxbe+At4uVnTuc6SYyOvRoblx/q6Ra6mdRipluMs7Go93eu/Oqo7cZ2akrNSi/WBXGqL/wXxe2pYsziSABWjQaqWXD/jKte1ZBx7OXssn88DUKat/RrL7Ak30WHzgY/goiNV+EPT/wtNYWHP3suAzzGqNNMhsPx0lSI7vSUga72J3hV1TOsX9ORosiRm1c2fnmNB9vJdH7yAX7lWpqH2XuOrldVvvTGazSJYx93eBukDViFCZ6egeb+gMT8VQ38uitwd9BZaMhycoujZgSjhx7JnbHUoUdXkdT3XBNV7fINTozWCNJW81iJN+JuLZzneJ/CbjmjkfGcVoFFPCD9rGSj8DbgvWpt8rkwI4ZfHTNOgO9iWgSklpk9r2wgM7Z87V2Zpu8UX/QCFxy8IREtJUo6X9YG9nBoy21wUa9zia2SWpideur9cXb6U0+q84onHVlA0Ehp/1OYs=
+    bucket: coveo-nprod-binaries
```

Comment:
We should adjust the bucket to `coveo-ndev-binaries` here to avoid mixing feature deployments with the official releases.

Same comment for line 111 holding the nightly build.

Small note is these buckets will be accessible under `staticdev.cloud` instead of `static.cloud`, which I think is good.

Reply:
@olamothe mentioned that, to make it simpler, we should push all of those to the prod CDN. Perhaps we need to have a discussion on it?

Reply:
My thinking behind the comment was to keep the prod bucket tidy by just holding official releases, and to keep the traffic separated.

I think it's good to keep our sandbox from sharing resources with our production files. We can delete everything (intentionally/unintentionally) without fear. In the prod bucket, our permissions are restricted, rightly so.

Reply:
I agree, I'd expect nightly and feature builds to be in the dev bucket.

Reply:
Yes, dev bucket is perfectly fine.

When I initially opened this JSUI ticket and we had the discussion many months ago, there was no bucket/CDN in dev/qa/prod.

But @samisayegh  did the work of terraforming those since then.

Reply:
And for dev, the encrypted `secret_access_key` you are using won't work. 

We will need to generate a new one for dev account that can only do this operation (push to this particular bucket) in AWS.

Reply:
Awesome, I'll do that, then.